vue/components/ui/molecules/lightbox/lightbox.vue

```diff
@@ -107,6 +115,14 @@ export const Lightbox = {
         close() {
             this.$emit("close");
         }
+    },
+    computed: {
+        classes() {

```

Comment:
Make this `imageStyle` instead and apply the `object-fit` directly, so the user can specify other values when needed.